landoapi/uplift.py

```diff
@@ -140,6 +154,13 @@ def get_uplift_conduit_state(
             f"Cannot create uplift for stack > {MAX_UPLIFT_STACK_SIZE} revisions."
         )
 
+    missing_bugs = get_revisions_without_bugs(phab, stack_data.revisions)
+    if missing_bugs:
+        missing = ", ".join(missing_bugs)
+        raise ValueError(
+            f"Every uplifted patch must have an associated bug ID: {missing} do not."
```

Comment:
I think it may be generally useful to have revision IDs (perhaps with PHIDs if needed) here, as it's not straightforward to look up revisions with a PHID, if one is trying to do some debugging.